scipy/stats/_stats_py.py

```diff
@@ -10314,3 +10314,108 @@ def first_order(t):
     # finding a wrong root.
     res = root_scalar(first_order, x0=x0, x1=x1)
     return res.root
+
+
+def ChebyMeanTest_1samp(a, popmean, confidence_level=0.95,
+                        axis=0, nan_policy='propagate'):
```

Comment:
`confidence_level` should be a method of the return object. See other tests such as `fligner` for e.g.